common/prometheus-monitors/templates/podmonitor.yaml

```diff
@@ -32,33 +32,10 @@ spec:
   namespaceSelector:
     matchNames:
 {{  default (list .Release.Namespace ) .Values.namespaces | toYaml | indent 6 }}
-  
+
   podMetricsEndpoints:
-    - interval: {{ default "60s" .Values.scrapeInterval }}
-      scrapeTimeout: {{ default "55s" .Values.scrapeTimeout }}
-      port: {{ default "metrics" .Values.metricsPort }}
```

Comment:
I have templated this once. There seems to be something wrong with the yaml rendering. 
<img width="785" alt="Screenshot 2024-03-06 at 15 05 58" src="https://github.com/sapcc/helm-charts/assets/56597015/5168f96e-35bc-4bcc-8bf6-b03c86774034">

Reply:
Thanks, there was still some diff between how the template was include in pod vs serviceMonitior